pkg/gofr/gofr.go

```diff
@@ -244,6 +254,17 @@ func (a *App) Shutdown(ctx context.Context) error {
 	return err
 }
 
+func (*App) isPortAvailable(port int) bool {
+	conn, err := net.DialTimeout("tcp", fmt.Sprintf(":%d", port), checkPortTimeout)
+	if err != nil {
```

Comment:
What about this `fmt.Sprint(port)` ?

Reply:
there is a colon ...I don't think just port works ...as that is not a valid format.